akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -538,33 +538,37 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
   }
 
   private def runInboundStreams(): Unit = {
-    runInboundControlStream()
-    runInboundOrdinaryMessagesStream()
+    val noCompression = new NoInboundCompression(system)
```

Comment:
Adding TODO, about making it used in all channels (AFAIR thats what we agreed a bit back), as next step